spikeinterface/preprocessing/filter_gaussian.py

```diff
@@ -0,0 +1,83 @@
+from typing import Iterable, Union
+import numpy as np
+from scipy.stats import norm
+from spikeinterface.core import BaseRecording, BaseRecordingSegment, get_chunk_with_margin
+from spikeinterface.core.core_tools import define_function_from_class
+from .basepreprocessor import BasePreprocessor, BasePreprocessorSegment
+
+
+class GaussianFilterRecording(BasePreprocessor):
```

Comment:
```suggestion
class GaussianBandpassFilterRecording(BasePreprocessor):
```
Can you rename the class for consistency?

(Sorry for being so picky :P)

Reply:
Of course, no problems!
Done :)